spark3-extensions/src/main/scala/org/apache/spark/sql/catalyst/parser/extensions/IcebergSparkSqlExtensionsParser.scala

```diff
@@ -163,9 +163,9 @@ case object IcebergSqlExtensionsPostProcessor extends IcebergSqlExtensionsBaseLi
   }
 
   private def replaceTokenByIdentifier(
-      ctx: ParserRuleContext,
-      stripMargins: Int)(
-      f: CommonToken => CommonToken = identity): Unit = {
+                                        ctx: ParserRuleContext,
+                                        stripMargins: Int)(
+                                        f: CommonToken => CommonToken = identity): Unit = {
```

Comment:
Can you revert this whitespace-only change?

Reply:
Woops, not sure when that snuck in

Reply:
Fixed!